retreiveUUID.py

```diff
@@ -0,0 +1,3 @@
+raw = input() # copy the line right under achievements
```

Comment:
Python, really? You couldn't have made this part in JavaScript too?

Reply:
ok no python for you then